### PR TITLE
Mvn aws id fixed 2

### PR DESCRIPTION
**Pull request by anuragtr**
- maven aws_id corrected in s3_helper
- few logs added

### CODE REVIEW

**Comment by deepak1725** [comment]
[test]

**Comment by deepak1725** [comment]
[test]

File: f8a_report/report_helper.py [hunk 1]
```diff
@@ -183,6 +183,10 @@ def invoke_emr_api(self, bucket_name, ecosystem, data_version, github_repo):
             'data_version': data_version
         }
 
+        logger.info('bucket_name for {eco}: {buck}'.format(eco=ecosystem, buck=bucket_name))
+        logger.info('github_repo for {eco}: {git}'.format(eco=ecosystem, git=github_repo))
+        logger.info('data_version for {eco}: {data}'.format(eco=ecosystem, data=data_version))
+
         try:
             # Invoke EMR API to run the retraining
             resp = requests.post(url=self.emr_api + '/api/v1/runjob', json=payload)
```
Comment (miteshvp) [inline f8a_report/report_helper.py]: It's also better to have `logger.info(resp.json())`

**Comment by anuragtr** [comment]
adding another log and raising a fresh PR

**Pull request closed without merging**